nautobot/extras/tests/test_dynamicgroups.py

```diff
@@ -667,6 +667,20 @@ def test_set_filter(self):
         # Cleanup because we're using class-based fixtures in `setUpTestData()`
         group.refresh_from_db()
 
+    def test_set_filter_on_ipaddress_dp(self):
+        """
+        Test `DynamicGroup.set_filter()` for an IPAddress Dynamic Group.
+        https://github.com/nautobot/nautobot/issues/6805
+        """
+        ipaddress_dp = DynamicGroup.objects.create(
```

Comment:
out of curiosity, should `_dp` be `_dg`, for a DynamicGroup?

Reply:
Yes, I do not know why I name it dp. I think too much dynamic programming lol